### DISCUSSION TITLE
Grid does not work using React library.

### DISCUSSION THREAD

**Discussion by digitaljohn** [nolimits4web/swiper#5257] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/owkwb

### Bug description

Same bug as reported here: #5157

Slides Grid does not work in React the official CodeSandbox is broken too, I can't find what exactly the problem is but the Core demo in CodeSandbox is working fine, but React is broken, even it is the same as Core.

### Expected Behavior

Working Slides Grid in React

### Actual Behavior

https://codesandbox.io/s/owkwb

### Swiper version

7.3.1

### Platform/Target and Browser Versions

macOS - Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
```
.swiper-slide {
  height: calc((100% - 30px) / 2) !important;
}
```
